clients.json

```diff
@@ -1964,5 +1964,14 @@
     "description": "Simple Redis Client for Dart",
     "authors": ["cerussite127"]
   }
+  
+  {
```

Comment:
Comma is missing after the previous client. That's why there is a JSON parse error and build failure.

Move this to `tools.json`, then I think we can merge it.